<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>425</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>425</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>calculator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/img/images/calculator_icon.png</normaloff>:/img/images/calculator_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
border:none;
background-color: rgb(238, 236, 236);
}

QPushButton 
{	
background-color:rgb(243,243,243);
}
QPushButton:hover 
{	
border:1px solid rgb(193,193,193);
background-color:rgb(221,223,221);
}

QPushButton#btn_numSign, #btn_num0, #btn_numDot, #btn_num1, #btn_num2, #btn_num3, #btn_num4, #btn_num5, #btn_num6, #btn_num7, #btn_num8, #btn_num9 {
    background-color: rgb(252, 252, 252);
}
QPushButton#btn_num0, #btn_num1, #btn_num2, #btn_num3, #btn_num4, #btn_num5, #btn_num6, #btn_num7, #btn_num8, #btn_num9 {
    font: bold 12pt '微软雅黑';
    background-color: rgb(252, 252, 252);
}
/*
//鼠标悬停时的样式
QPushButton#btn_numSign:hover,#btn_num0:hover,#btn_numDot:hover,#btn_num1:hover,#btn_num2:hover,#btn_num3:hover,#btn_num4:hover,#btn_num5:hover,#btn_num6:hover,#btn_num7:hover,#btn_num8:hover,#btn_num9:hover
{	
border:1px solid rgb(193,193,193);
background-color:rgb(221,223,221);
}
*/
/* 鼠标按下时的样式 */
QPushButton#btn_num0:pressed,
QPushButton#btn_num1:pressed,
QPushButton#btn_num2:pressed,
QPushButton#btn_num3:pressed,
QPushButton#btn_num4:pressed,
QPushButton#btn_num5:pressed,
QPushButton#btn_num6:pressed,
QPushButton#btn_num7:pressed,
QPushButton#btn_num8:pressed,
QPushButton#btn_num9:pressed 
{
    background-color: #ddd;  /* 鼠标按下时的背景颜色 */
}
/* 禁用状态下的样式 */
QPushButton:disabled
 {
    background-color: #eee;  /* 禁用时的背景颜色 */
    color: #888;  /* 禁用时的文本颜色 */
}
QLineEdit#lineEdit_vice {
    color: #808080; /* 深灰色 */
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLineEdit" name="lineEdit_vice">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit_main">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="3" column="0">
      <widget class="QPushButton" name="btn_num4">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="btn_num3">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="btn_add">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btn_multi">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="btn_equal">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btn_num9">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btn_num5">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btn_div">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>÷</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btn_del">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>del</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="btn_numSign">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btn_num7">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn_reciprocal">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>1/x</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_C">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btn_square">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>x²</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btn_num8">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn_squareRoot">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>2√x</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btn_num2">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="btn_num1">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_percentage">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="btn_numDot">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btn_num6">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="btn_sub">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_CE">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>CE</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="btn_num0">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
